index.js

```diff
@@ -321,117 +309,38 @@ const HomeNode = module.exports = {
-    });
-
-    startupSequence = startupSequence.then(() => {
-      return Datastore.startup();
-    });
-
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Datastore restore complete.');
-    });
-
-    // Restore Device Traits
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Restoring device traits...');
-    });
-
-    startupSequence = startupSequence.then(() => {
-      _.each(HomeNode.instances.devices, (device) => {
-        device.restoreTraits();
-      });
-    });
-
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Device traits restored.');
-    });
-
-    // Start Interfaces
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Starting Interfaces...');
-    });
-
-    startupSequence = _.reduce(HomeNode.instances.interfaces, (promiseChain, instance, id) => {
-      return promiseChain.then(() => {
-        SysLogger.log(`Starting interface: ${id}`);
-        return instance.startup();
-      });
-    }, startupSequence);
-
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Interfaces startup complete.');
-    });
-
-    // Start Devices
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Starting Devices...');
-    });
-
-    startupSequence = _.reduce(HomeNode.instances.devices, (promiseChain, instance, id) => {
-      return promiseChain.then(() => {
-        SysLogger.log(`Starting device: ${id}`);
-        return instance.startup();
-      });
-    }, startupSequence);
-
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Devices startup complete.');
-    });
-
-    // Start polling devices
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Starting polling on devices...');
-    });
-
-    startupSequence = _.reduce(HomeNode.instances.devices, (promiseChain, instance, deviceId) => {
-      // Step into polling on each device
-      return promiseChain.then(() => {
-        return _.reduce(instance.polling, (promiseChain, poll, pollId) => {
-          return promiseChain.then(() => {
-            SysLogger.log(`Registering polling (${pollId}) on device (${deviceId})`);
-
-            // Register poll
-            setInterval(() => {
-              instance.runPoll(pollId);
-            }, poll.secs * 1000);
-
-            // Optionally run at startup
-            if (poll.runAtStartup) {
-              return instance.runPoll(pollId);
-            }
-          });
-        }, promiseChain);
-      });
-    }, startupSequence);
-
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Devices polling setup complete.');
-    });
-
-    startupSequence = startupSequence.then(() => {
-      SysLogger.log('Starting automations...');
-    });
-
-    startupSequence = startupSequence.then(() => {
-      _.each(HomeNode.instances.automations, (automation, id) => {
-        SysLogger.log(`Starting automation: ${id}`);
```

Comment:
Another log, that provides some visibility into the startup process. Should we keep it?